3-0-spring-mvc/src/main/java/com/bobocode/mvc/controller/NoteRESTController.java

```diff
@@ -0,0 +1,26 @@
+package com.bobocode.mvc.controller;
+
+import com.bobocode.mvc.model.Note;
+import com.bobocode.mvc.service.NoteService;
+import lombok.Data;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Data
+@RestController
+@RequestMapping("api/v1/notes")
+public class NoteRESTController {
```

Comment:
@erneman1 let's move it to package `api` instead of controller. Also let's call it `NoteRestController`